qoqo-qryd/src/tweezer_devices.rs

```diff
@@ -147,7 +147,11 @@ impl TweezerDeviceWrapper {
     /// Returns:
     ///     str: The name of the current layout.
     pub fn current_layout(&self) -> &str {
-        self.internal.current_layout.as_str()
+        self.internal
+            .current_layout
+            .as_ref()
+            .expect("None")
```

Comment:
Do I understand it correctly that it is basically guaranteed that this will never happen? In that case we should have a more detailed errror message like "Internal error, current layout unexpectedly None". Or we should raise a PyRuntimeError.

Reply:
Just something like "Current layout not set" should be fine.